contributors/devel/sig-architecture/godep.md

```diff
@@ -1,268 +1,2 @@
-`git commit` which includes just the dependencies and Godeps files, and
-another `git commit` that includes changes to Kubernetes code to use (or stop
-using) the new/updated/removed dependency.  These commits can go into a single
-pull request.
-
-Before sending your PR, it's a good idea to sanity check that your
-Godeps.json file and the contents of `vendor/ `are ok:
-
-```sh
-hack/run-in-gopath.sh hack/verify-godeps.sh
-```
-
-All this script will do is a restore, followed by a save, and then look for
-changes.  If you followed the above instructions, it should be clean.  If it is
-not, you get to figure out why.
-
-## Manual updates
-
-It is sometimes expedient to manually fix the `Godeps.json` file to
-minimize the changes. However, without great care this can lead to failures
-with the verifier scripts.  The kubernetes codebase does "interesting things"
-with symlinks between `vendor/` and `staging/` to allow multiple Go import
-paths to coexist in the same git repo.
-
-The verifiers, including `hack/verify-godeps.sh` *must* pass for every pull
-request.
-
-## Reviewing and approving dependency changes
-
-Particular attention to detail should be exercised when reviewing and approving
-PRs that add/remove/update dependencies. Importing a new dependency should bring
-a certain degree of value as there is a maintenance overhead for maintaining
-dependencies into the future.
-
-When importing a new dependency, be sure to keep an eye out for the following:
-- Is the dependency maintained?
-- Does the dependency bring value to the project? Could this be done without
-  adding a new dependency?
-- Is the target dependency the original source, or a fork?
-- Is there already a dependency in the project that does something similar?
-- Does the dependency have a license that is compatible with the Kubernetes
-  project?
-
-Additionally:
-- Look at the godeps file. Check that the only changes are what the PR claims
-  them to be. 
-- Check if there is a tagged release we can vendor instead of a random hash
-- Scan the imported code for things like init() functions
-- Look at the Kubernetes code changes and make sure they are appropriate
-  (e.g. renaming imports or similar). You do not need to do feature code review.
-- If this is all good, approve, but don't LGTM, unless you also do code review
-  or unless it is trivial (e.g. moving from k/k/pkg/utils -> k/utils).
-
-All new dependency licenses should be reviewed by @kubernetes/dep-approvers to ensure that they
-are compatible with the Kubernetes project license. It is also important to note
-and flag if a license has changed when updating a dependency, so that these can
-also be reviewed.
-
-For reference, whitelisted licenses as per the CNCF Whitelist Policy are
-mentioned [here](https://git.k8s.io/sig-release/licensing/README.md#licenses-for-dependencies).
+See [current documentation for working with dependencies](./vendor.md) for master branch development.
```

Comment:
we could also just delete the file entirely but this seems like a reasonable "redirect" ...